be/src/olap/rowset_builder.cpp

```diff
@@ -368,6 +369,11 @@ void BaseRowsetBuilder::_build_current_tablet_schema(int64_t index_id,
     _tablet_schema->set_table_id(table_schema_param->table_id());
     // set partial update columns info
     _partial_update_info = std::make_shared<PartialUpdateInfo>();
+    DBUG_EXECUTE_IF(
+            "BaseRowsetBuilder::_build_current_tablet_schema.check_partial_update_column_num", {
+                EXPECT_EQ(table_schema_param->partial_update_input_columns().size(),
```

Comment:
It's better to add the debug point before calling `_build_current_tablet_schema`, check the partial update input columns of `_req.table_schema_param`, if not match, return an error code. Then you don't need to import gtest dependency here.